src/mono/mono/metadata/class.c

```diff
@@ -5855,6 +5855,26 @@ is_valid_family_access (MonoClass *access_klass, MonoClass *member_klass, MonoCl
 	return TRUE;
 }
 
+static gboolean
+ignores_access_checks_to (MonoAssembly *accessing, MonoAssembly *accessed)
+{
+	if (accessing == accessed)
+		return TRUE;
+	if (!accessing || !accessed)
+		return FALSE;
+
+	mono_assembly_load_friends (accessing);
+	for (GSList *tmp = accessing->ignores_checks_assembly_names; tmp; tmp = tmp->next) {
+		MonoAssemblyName *victim = (MonoAssemblyName *)tmp->data;
+		if (!victim->name)
+			continue;
+		if (g_ascii_strcasecmp (accessed->aname.name, victim->name))
```

Comment:
Purely a nit, but isn't this cleaner by just writing `if (!condition) return`? The continue seems odd.